.github/workflows/vcpkg_ci_amd64.yml

```diff
@@ -38,8 +38,9 @@ jobs:
       fail-fast: false
       matrix:
         image:
-          - { name: 'ubuntu-v2', tag: '20.04' }
-          - { name: 'ubuntu-v2', tag: '22.04' }
+          # 'name' is Docker image name whereas 'os' is more generic
+          - { os: 'ubuntu', name: 'ubuntu-v2', tag: '20.04' }
```

Comment:
is it still useful to have the name here if we're just going to use `matrix.image.os`?

Reply:
We use `name` for the docker container that we run the steps in and also in the caches